README.md

```diff
@@ -140,6 +140,9 @@ Write your test in a new or existing `test_*.py` file under `$PROJECT_HOME/firef
 * [Marionette docs][marionette-docs]
 * [Firefox Puppeteer docs][ff-puppeteer-docs]
 
+Special Requirements
+--------------------
+In order to run EME tests, you must use a profile that has logged into the EME provider and save the credentials.
```

Comment:
Also explain that a custom ini file of urls is needed and provide an example of how to invoke from command-line. Pointing readers to https://developer.mozilla.org/en-US/docs/Mozilla/Command_Line_Options and/or https://developer.mozilla.org/en-US/docs/Profile_Manager would be lovely.